packages/common/src/FormattingUtils.js

```diff
@@ -117,6 +119,30 @@ function addSign(number) {
   }
 }
 
+// Take an amount based on fromDecimals and convert it to an amount based on toDecimals
+// For example amount = 100 usdt, its decimals are 6. You want to convert it to 18.
+// convertDecimals(6,18)(100000000)  => 1000000000000000000000.
+// Returns a BigNumber you will need to call toString on
+// fromDecimals: number - decimal value of amount
+// toDecimals: number - decimal value to convert to
+// bn: toBN function - optionally provide your own BN function.
+// return => (amount:string)=>BN
+const ConvertDecimals = (fromDecimals, toDecimals, bn = toBN) => {
+  assert(fromDecimals >= 0, "requires fromDecimals as an integer >= 0");
```

Comment:
this function came back because its used in the emp client. Theres[ one call ](https://github.com/UMAprotocol/protocol/pull/1913/files#diff-d204664e0a1e6715bec3ac212d501fc0R235) where you need to convert from sythetic decimals to collateral decimals. This functionality is not supported by ethers parseFixed afaik.